develop-guide/writeblitz/writeblitz.rst

```diff
@@ -0,0 +1,759 @@
+.. _write-blitz:
+
+Write a BLITZ
+-------------
+
+The *BLITZ* also known as *Quick Trigger* is a YAML-driven template that makes it easy for you to run
+a test case without having to know any knowledge of programming. The *Quick Trigger* ---
+called *BLITZ* because it's lightning fast --- does the following actions:
+
+* Configure a device.
+* Parse the device output to verify if the device state is as expected.
+* Unconfig or modify the initial configuration.
+* Parse the output to check the operational state.
+* Learn a feature and verify the result of the action
+* Calling different apis and use their outputs on other actions and other devices
+* Yang integration
+* It is fully customizable and new actions can be added
```

Comment:
add a Many more, the full list would be below

Reply:
Done